nsls2tools/csx1/startup/__init__.py

```diff
@@ -0,0 +1,11 @@
+from .startup import *
+from .accelerator import *
+from .optics import *
+from .endstation import *
+from .detectors import *
+#from ..bluesky.xpcs import *
```

Comment:
Intentionally commented out? If so please remark on why.

Reply:
We need to test this plan as it was something we did before the gs.DETS were removed. I believe it should be okay, but I think we might want to add some metadata to determine the state of the beam block and gain state.